upgrade_ksql.yml

```diff
@@ -89,7 +89,7 @@
         state: absent
       when:
         - ansible_os_family == "Debian"
-        - not confluent-server_enabled|bool
+        - not confluent_server_enabled|bool
```

Comment:
i think this change also needs to be made in tasks/upgrade_component.yml

Reply:
Good catch!